velox/common/file/FileSystems.h

```diff
@@ -43,6 +43,11 @@ struct FileOptions {
 
   std::unordered_map<std::string, std::string> values;
   memory::MemoryPool* pool{nullptr};
+  std::optional<int64_t> fileSize{std::nullopt};
```

Comment:
This doesn't quite feel like an option. Would you add a comment to explain that if this is specified, then this value can be trusted?

Reply:
Added a comment.